src/middlewared/middlewared/alembic/versions/22.12/2022-05-27_12-30_cpu_pinning.py

```diff
@@ -0,0 +1,28 @@
+"""CPU Pinning
+
+Revision ID: 0267cef97ec3
+Revises: 59fbc9897ec3
+Create Date: 2022-05-27 12:30:03.593598+00:00
+
+"""
+from email.policy import default
```

Comment:
What is this for?

Reply:
I forgot to remove it. I created this migration manually by copying existing one since I wasn't able to get alembic create one for me. Could that possibly cause some problems? Also on that note is there some dev vm/container which has a proper python/tooling setup for this project which can be used?

Reply:
Your migration is correct.

We don't have any special dev images, you can just install the necessary tools using `apt` (since TrueNAS is just a Debian Bullseye). To update the code we usually just rsync `/usr/lib/python3/dist-packages/middlewared` or mount it from developer machine with NFS. I can build a custom ISO image from this PR if you like, but if you were able to fully test it by updating usual Nightly build python files and restarting middleware, that should not be necessary. Thank you for your contribution!

Reply:
Thank you but no need for that iso like you said. I was using the Makefile to sync code into my Truenas machine and it worked just fine. It was just the alembic command that didn't work when I tried to create new migration file like instructed in the docs. Maybe there's something wrong in my local python setup but if the manually created file is fine then I'm good. =)